src/GitHub.App/ViewModels/RepositoryPublishViewModel.cs

```diff
@@ -157,18 +150,14 @@ IObservable<ProgressState> OnPublishRepository(object arg)
             var account = SelectedAccount;
 
             return repositoryPublishService.PublishRepository(newRepository, account, SelectedHost.ApiClient)
-                .Select(_ =>
-                {
-                    vsServices.ShowMessage("Repository published successfully.");
-                    return ProgressState.Success;
-                })
+                .Select(_ => ProgressState.Success)
```

Comment:
This no longer shows the "Repository published successfully." message that used to happen in this spot via the old call to `vsServices.ShowMessage("Repository published successfully.");`. Was that intentional?